Automattic/sensei
# Add margin to thumbnails in course listing

Fixes #2276

#### Changes proposed in this Pull Request:

* Add right margins to thumbnails in course listing
* Should also work with lesson thumbnails

#### Testing instructions:

* View Courses page in Storefront theme
* Experience proper thumbnail alignment
* Check that other themes also continue working

#### Screenshot / Video

![image](https://user-images.githubusercontent.com/176949/74527002-f1100f80-4f24-11ea-9660-e05b7b0d9c29.png)



#### Proposed changelog entry for your changes:

* Improve Storefront theme compatibility - Course list thumbnail alignment
